From b164f54f8bb274db0197e0c1aff6d5ce1431d4f0 Mon Sep 17 00:00:00 2001
From: Sai Vemuri <svemuri@redhat.com>
Date: Wed, 12 Nov 2014 19:08:15 -0500
Subject: [ethernet] cxgb4vf: Replace repetitive pci device ID's with right ones

Message-id: <1415819300-32670-15-git-send-email-svemuri@redhat.com>
Patchwork-id: 99870
O-Subject: [RHEL7.1 PATCH BZ 1163467 14/19] cxgb4vf: Replace repetitive pci device ID's with right ones
Bugzilla: 1163467
RH-Acked-by: Jes Sorensen <Jes.Sorensen@redhat.com>
RH-Acked-by: David S. Miller <davem@redhat.com>

Replaced repetive Device ID's which got added in commit b961f9a48844ecf3
("cxgb4vf: Remove superfluous "idx" parameter of CH_DEVICE() macro")

Signed-off-by: Hariprasad Shenai <hariprasad@chelsio.com>
Signed-off-by: David S. Miller <davem@davemloft.net>
(cherry picked from commit cd03cf0158449f9f4c19ecb54dfc97d9bd86eeeb)
Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/drivers/net/ethernet/chelsio/cxgb4vf/cxgb4vf_main.c b/drivers/net/ethernet/chelsio/cxgb4vf/cxgb4vf_main.c
index 51d1369..f113821 100644
--- a/drivers/net/ethernet/chelsio/cxgb4vf/cxgb4vf_main.c
+++ b/drivers/net/ethernet/chelsio/cxgb4vf/cxgb4vf_main.c
@@ -2929,14 +2929,14 @@ static const struct pci_device_id cxgb4vf_pci_tbl[] = {
  CH_DEVICE(0x480d), /* T480-cr */
  CH_DEVICE(0x480e), /* T440-lp-cr */
  CH_DEVICE(0x4880),
- CH_DEVICE(0x4880),
- CH_DEVICE(0x4880),
- CH_DEVICE(0x4880),
- CH_DEVICE(0x4880),
- CH_DEVICE(0x4880),
- CH_DEVICE(0x4880),
- CH_DEVICE(0x4880),
- CH_DEVICE(0x4880),
+ CH_DEVICE(0x4881),
+ CH_DEVICE(0x4882),
+ CH_DEVICE(0x4883),
+ CH_DEVICE(0x4884),
+ CH_DEVICE(0x4885),
+ CH_DEVICE(0x4886),
+ CH_DEVICE(0x4887),
+ CH_DEVICE(0x4888),
  CH_DEVICE(0x5801), /* T520-cr */
  CH_DEVICE(0x5802), /* T522-cr */
  CH_DEVICE(0x5803), /* T540-cr */
-- 
1.7.1